src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,24 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveArtworkItem: {
+      type: ConversationItemType,
+      resolve: (artworkItem, _args, { artworkLoader }) => {
```

Comment:
I changed the name of this from `parent` to `artworkItem` to be more explicit. Curious of your thoughts on this @starsirius?

Reply:
Non-blocking but I think `conversationItem` might be better name, given that it could be a partner show (and we are actually checking the type below).